### DISCUSSION TITLE
Help with installing DoVi_Script

### DISCUSSION THREAD

**Discussion by jangomunkie** [R3S3t9999/DoVi_Scripts#21] [Q&A]
Hi.
I've just installed DoVi Script and the pre-requists as per the instructions but I must've done something wrong as I get a list of errors when trying to convert a DV profile 7 MKV to profile 5.  They flash up so quickly and then closes the .bat that I can't list all the details of all the *.exe listed but they are along the line of:
tools\mediainfo.exe is not recognized as an internal or external command, operable program or batch file.

What have I done wrong?

**Comment by R3S3t9999** [comment]
No idea but make sure you dont have special characters or space in your dovi_script folder and path. This is the most common source of problems. Also avoid using the script on an OS drive (C:\)

**Reply by jangomunkie** [reply]
Hi thanks for the quick reply. There are no special characters in the path.  I installed the script folder path on my D: drive.  I only have one drive in my laptop so the system drive (C:) and the D: drive are partitions on the same physical disk, does this matter?

**Reply by R3S3t9999** [reply]
must not have any space in the path too. see installation instructions: https://www.youtube.com/watch?v=6SLtsVDse2w
Also, your original question is P7 To P5 but I guess you meant P8 because you cannot convert P7 to P5.
